apollo-gradle-plugin/src/test/testProject/android/buildScriptFixtures/pojo.gradle

```diff
@@ -0,0 +1,50 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:2.3.0-beta2'
+  }
+}
+
+plugins {
+  id 'com.android.application'
+  id 'com.apollographql.android'
+}
+
+android {
+  compileSdkVersion 25
+  buildToolsVersion '25.0.2'
+
+  defaultConfig {
+    applicationId "com.example.apollographql.testProject.basic"
+    minSdkVersion 15
+    targetSdkVersion 25
+  }
+
+  lintOptions {
+    textReport true
+    textOutput 'stdout'
+    ignore 'InvalidPackage', 'GoogleAppIndexingWarning', 'AllowBackup'
+  }
+}
+
+repositories {
+  jcenter()
+  maven { url "https://jitpack.io" }
+  maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+}
+
+dependencies {
+  compile 'com.android.support:appcompat-v7:25.1.0'
+  compile 'com.apollographql.android:api:0.0.1-SNAPSHOT'
+}
+
+apollo {
+  generateClasses = true
+  customTypeMapping {
+    DateTime = "java.util.Date"
```

Comment:
@digitalbuddha ok I added the case with Optional the only for now we don't support mapping to classes with generics so we can have only this:
```
apollo {
  generateClasses = true
  customTypeMapping {
    DateTime = "java.util.Date"
    OptionalString = "com.fernandocejas.arrow.optional.Optional"
  }
}
```
but I've created an issue to add such support #135